### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by Sven-Lyco**
Restructuring CSS-Files and editing index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,28 +20,28 @@
...
         </figure>
       </div>
 
       <!--Navigation-->
       <nav class="navigation" id="top">
         <a class="button" href="#biography">About Me</a>
-        <a class="button" href="#contact">Contact</a>
         <a class="button" href="#socials">Socials</a>
         <a class="button" href="#cv">CV</a>
+        <a class="button" href="#contact">Contact</a>
```
Comment (macfrei) [inline index.html:36]: If you don't use the `button` class elsewhere, you could think about naming these after the BEM convention. In this case how about: `navigation__link`?

File: style/components/footer.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  padding: 10px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 10px;
+  justify-content: flex-start;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  box-shadow: -3px -8px 10px #363537;
+}
```
Comment (macfrei) [inline style/components/footer.css:14]: To be more specific, this could be a class: `.footer`

**Review by macfrei — COMMENTED** [review]
Hi Sven, you did a nice job here! 👍

File: index.html (hunk 1 above)
Comment (Sven-Lyco) [reply index.html:36]: First I had the same `button` at the end of my page to go back to top. But i changed it to an other fixed button. So i will change these buttons it according to your suggestion.

**Comment by Sven-Lyco** [comment]
@macfrei 
I have made the changes according to your comments.
Do you want to check it again or can i merge pull request ?

**Comment by macfrei** [comment]
Yes you can :)

**Comment by Sven-Lyco** [comment]
> Yes you can :)

Thank you very much and enjoy your evening!

**Pull request merged**